script.js

```diff
@@ -1,20 +1,427 @@
+// -----------ADD FUNCTION TO MOBILE MENU BAR------------
+
 const hamburger = document.querySelector('.hamburger');
 const mobileMenuBar = document.querySelector('.hamburger-menu-container');
 const navLinks = document.querySelectorAll('.nav-link');
 
-function mobileMenu() {
+const mobileMenu = () => {
   hamburger.classList.toggle('active');
   mobileMenuBar.classList.toggle('show-menu');
-}
+};
 
 hamburger.addEventListener('click', mobileMenu);
 
-function removeMenu() {
+const removeMenu = () => {
   mobileMenuBar.classList.remove('show-menu');
   hamburger.classList.remove('active');
-}
+};
 const navLinksArray = Array.from(navLinks);
-// eslint-disable-next-line no-plusplus
-for (let i = 0; i < navLinksArray.length; i++) {
+for (let i = 0; i < navLinksArray.length; i += 1) {
   navLinksArray[i].addEventListener('click', removeMenu);
 }
+
+// -----------REFACTOR PROJECT SECTION----------------
+
+// Store project information in array
+const projectArray = [
+  {
+    projectHeading: 'Multi-Post Stories Gain+Glory',
+    projectName: 'Keeping track of hundreds of components',
+    projectDescription: `Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a galley of
+          type and scrambled it 1960s with the releaLorem Ipsum is simply
+          dummy text of the printing and typesetting industry. Lorem Ipsum
+          has been the industry's standard dummy text ever since the 1500s,
+          when an unknown printer took a galley of type and scrambled it
+          1960s with the release`,
+    projectImageMobile: './images/popup-mobile-img.svg',
+    projectImageDesktop: './images/popup-desktop-img.svg',
+    technologies1: 'Ruby on rails',
+    technologies2: 'css',
+    technologies3: 'JavaScript',
+    technologies4: 'Codekit',
+    technologies5: 'GitHub',
+    technologies6: 'Bootstrap',
+    technologies7: 'Terminal',
+    technologies8: 'Codepen',
+    technologies9: 'html',
```

Comment:
- [ ] **[OPTIONAL]:** I see that you did a great job using an array of project objects! 🚀 However, naming technologies one by one is not very maintainable. Let's say you have a project on which you use 20+ Technologies, your object would have 20+ different technologies properties. For this specific case, it would be great to use an Array which you can iterate and render as many technology badges as the project have. Please consider updating this! 🤓 🛠️

Reply:
You are very right. I would improve on this code and make it DRY. Thanks for the review